File: app/src/main/java/com/aceplus/hackthon/MyApp.java
package com.aceplus.hackthon;

import android.app.Application;

import com.aceplus.shared.model.BackendModel;

/**
 * Created by kyawthetwin on 9/22/18.
 */

public class MyApp extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        BackendModel.Companion.initBackendModel(getApplicationContext());
    }
}

File: backend/src/main/java/com/aceplus/backend/MyApp.kt
package com.aceplus.backend

import android.app.Application
import com.aceplus.shared.model.BackendModel

/**
 * Created by kkk on 9/22/2018.
 */
class MyApp : Application() {
    override fun onCreate() {
        super.onCreate()
        BackendModel.initBackendModel(applicationContext)
    }
}